Detections/MultipleDataSources/ZincJan272021IOCs.yaml

```diff
@@ -50,21 +50,24 @@ query: |
   | parse Message with * '(' DNSName ')' * 
   | where isnotempty(FileHash)
   | where FileHash in~ (SHA256Hash) or DNSName in~ (DomainNames)
-  | extend Account = SourceUserID, Computer = DeviceName, IPAddress = SourceIP
+  | extend DataType = "CommonSecurityLog", Account = SourceUserID, Computer = DeviceName, IPAddress = SourceIP
```

Comment:
You can simply use Type here, it is a field available on all data types and provides you with the name of it without having to manually map it.